packages/block-editor/src/components/use-block-sync/index.js

```diff
@@ -70,8 +79,10 @@ export default function useBlockSync( {
 	selectionEnd: controlledSelectionEnd,
 	onChange = noop,
 	onInput = noop,
+	entityId,
 } ) {
 	const registry = useRegistry();
+	const BlockManager = useBlockManager( entityId );
```

Comment:
I think this needs to switch to a lowercase first letter. It's neither a class nor a component.

Reply:
good idea. we could even destructure the functions here.